app/server.go

```diff
@@ -51,14 +54,23 @@ func (s *server) MockHandler(w http.ResponseWriter, r *http.Request, ps httprout
 
 	// Add any user defined headers
 	for k, v := range route.ResponseHeaders {
+		v, err := ctx.ReplaceVariables(v)
+		if err != nil {
+			log.Errorf("Error parsing variable %s - %s", v, err)
+			continue
+		}
 		w.Header().Set(k, v)
 	}
 
 	// Write out user defined response code
 	w.WriteHeader(route.ReturnCode)
 
 	// Write Body to caller
-	fmt.Fprintf(w, "%s", route.Body)
+	varBody, err := ctx.ReplaceVariables(route.Body)
+	if err != nil {
+		log.Errorf("Error parsing variable %s - %s", route.Body, err)
```

Comment:
Should we include the path in this log statement? See above logs using WithFields.